frontend/src/lib/components/widgets/Multiselect/Multiselect.tsx

```diff
@@ -270,6 +271,21 @@ class Multiselect extends React.PureComponent<Props, State> {
             filterOptions={this.filterOptions}
             closeOnSelect={false}
             overrides={{
+              SelectArrow: {
+                component: ChevronDown,
+
+                props: {
+                  overrides: {
+                    Svg: {
+                      style: () => ({
+                        width: "24px",
```

Comment:
nit: is it possible to create a variable in `sizes.ts` that is like `svgHeight: "24px"` or `"1.5rem"`? 

Or could we use `iconSizes.xl`?

Reply:
That's a great callout! Fixed on [34a0b0e](https://github.com/streamlit/streamlit/pull/6817/commits/34a0b0e3d4c89c39c33ff71ecba9498b746990bd)